tests/modeltests/tst_models.cpp

```diff
@@ -351,28 +406,11 @@ private slots:
         }
     }
 
-    void testSymbol()
+    void testPrettySymbol()
```

Comment:
please use QTestLib's data-driven tests instead. (cf. QFETCH, QTest::addColumn, QTest::addRow)

but I can clean this up for you ~next week if you don't feel like it

Reply:
Ah, now I see what you meant with your previous comment regarding the unit test. Didn't know about QFETCH and friends. I'll fix.